packages/edit-navigation/src/components/menu-editor/use-navigation-blocks.js

```diff
@@ -128,6 +132,10 @@ export default function useNavigationBlocks( menuId ) {
 		for ( const deletedClientId of deletedClientIds ) {
 			// TODO - delete menu items.
 		}
+
+		createSuccessNotice( __( 'Navigation saved.' ), {
+			type: 'snackbar',
+		} );
```

Comment:
In my testing this displays immediately instead of waiting for a successful save as this is a synchronous function. As mentioned in #21344, it's really hard to tell when a save has completed.

Lets merge anyway, but keep the issue open. I know there are other PRs like #22148 that would change this completely, so probably good to see some progress around that first.

Reply:
Yes, I think so as well, wrote in the description "Advances #21344" :) so we don't close it. We need more PRs for the other notifications as well (like location editing).

Thanks for reviewing and merging! 🙇